src/operation/overlayng/OverlayNGRobust.cpp

```diff
@@ -265,7 +265,7 @@ OverlayNGRobust::snapTolerance(const Geometry* geom)
 double
 OverlayNGRobust::ordinateMagnitude(const Geometry* geom)
 {
-    if (geom == nullptr) return 0;
```

Comment:
@dr-jts I fixed a couple issues in this commit where the meaningless placeholder values for a null Envelope are used in comparisons. The same issues may be present in JTS?

Reply:
Another one (addressed in https://github.com/libgeos/geos/pull/407/commits/f16218f1b13b1268be72788928ca1be037244a74) comes from creating an `QuadEdgeSubdivision` with a null envelope.

Reply:
This should never be called with an empty geometry, since that will not create a robustness failure.  But it's good to harden the code.